mercury_app/utils.py

```diff
@@ -18,15 +18,15 @@ def get_api_organization(token):
     Get organization from the user of the token from the api
     """
     eventbrite = Eventbrite(token)
-    return eventbrite.get('/users/me/organizations')['organizations']
+    return eventbrite.get('/users/me/organizations').get('organizations')
 
 
 def get_api_events_org(token, organization):
     """
     Get events of one organization from the user of the token from the api
     """
     eventbrite = Eventbrite(token)
-    return eventbrite.get('/organizations/{}/events/'.format(organization['id']))['events']
+    return eventbrite.get('/organizations/{}/events/'.format(organization['id'])).get('events')
```

Comment:
2 comments here:
- Why are you preferring `.get()` instead of accessing by key directly? AFAIK the key `events` will be present always, in case the organization does not have any event will be `[]`.
- The pagination is being discarded, what would happen if you have more than one page?